libclc/utils/libclc-remangler/LibclcRemangler.cpp

```diff
@@ -271,6 +285,7 @@ class Remangler {
       : AST(AST), Root(Root), TypeReplacements(TypeReplacements) {
     MangleContext.reset(
         ItaniumMangleContext::create(*AST, AST->getDiagnostics()));
+    TargetGenericAddrSpace = AST->getTargetAddressSpace(LangAS::Default);
```

Comment:
Should this be called `TargetDefaultAddrSpace`? I worry that 'generic' is an overloaded term.

Reply:
> Should this be called `TargetDefaultAddrSpace`? I worry that 'generic' is an overloaded term.

done, thanks for the suggestion